GSG_Cypress/cypress/e2e/OrangeHRM/leaveRequest.spec.cy.ts

```diff
@@ -0,0 +1,127 @@
+// import LoginPage from '../../support/pageObjects/loginPage'
+import { login, logout, requestLeave, approveReject } from '../../support/helpers/orangeHRM/leaveRequestHelper';
+import { checkDataInTable } from '../../support/utils/orangeHRMUtils/checkDataInTable'
+
+// const loginObj: LoginPage = new LoginPage();
+
+let empNumberResponse: number
+let username = `HaneenIbrahem${Math.floor((Math.random() * 1000))}`;
+let password = "haneen123";
+let vacancyId: number;
+
+describe('Leave Request Scenario', () => {
+  beforeEach(() => {
+    cy.fixture('addEmployee').as('data')
+    cy.fixture('leave').as('leaveData')
+    cy.visit("https://opensource-demo.orangehrmlive.com/web/index.php/auth/login");
+    login("Admin", "admin123");
+
+    cy.get('@data').then((infoData: any) => {
+      cy.request({
+        method: 'POST',
+        url: 'https://opensource-demo.orangehrmlive.com/web/index.php/api/v2/pim/employees',
```

Comment:
you can declare this https://opensource-demo.orangehrmlive.com/ as Base URL and just declare the rest of url you need